### PR TITLE
Enhance Documentation

### PR DESCRIPTION
**Pull request by Permanently**
Just to avoid #463 occurring again, here's a change to README.md to reflect the latest version's usage changes.

### CODE REVIEW

**Comment by OiYouYeahYou** [comment]
My preference would be to recommend adding the plugin config to the `.prettierrc`, rather than adding it to every command

**Comment by Permanently** [comment]
> My preference would be to recommend adding the plugin config to the `.prettierrc`, rather than adding it to every command

This was more for just the README.md for people's reference, but if it's applicable then might as well - do you mind referencing where? (can't seem to find anywhere I can add it to, and I'm at work)

**Comment by OiYouYeahYou** [comment]
Same place that documents the CLI option: https://prettier.io/docs/en/plugins#using-plugins

```json
{
  "plugins": ["@prettier/plugin-pug"]
}
```

**Comment by Permanently** [comment]
You did make me remember another file for editing, so caught that whilst it was in my head. Thanks for unintentionally reminding me.

Some people can just add it to the file, but for a CLI lover (like myself) I'd rather stick to having it in-line. You're right though, the other methods should also be covered. This PR was originally gonna cover just the method that's documented, but I'll have a look later and add that in if time's on my side. (if @Shinigami92 or anyone else doesn't do it before me)

**Comment by Shinigami92** [comment]
> You did make me remember another file for editing, so caught that whilst it was in my head. Thanks for unintentionally reminding me.

So is this ready to merge? Or do you need to do something else and right now you cant because e.g. you are out of home, touching grass?

**Review by Shinigami92 — DISMISSED** [review]

**Comment by Shinigami92** [comment]
> > do you need to do something else and right now you cant because e.g. you are out of home, touching grass?
> 
> You won't believe this fella. Grass is real. I thought it was a myth.

:exploding_head: :green_heart: 

> Yep, still a draft - lost the button to change the PR state back to draft. If you give me a few hours, I'll make the necessary commits to address the various other methods.

File: README.md [hunk 1]
```diff
@@ -61,13 +61,28 @@ pnpm add --save-dev prettier @prettier/plugin-pug
 ```
 
 ## Usage
-
+#### [CLI](https://prettier.github.io/plugin-pug/guide/#cli)
```
Comment (Shinigami92) [inline README.md:64]:
```suggestion

#### [CLI](https://prettier.github.io/plugin-pug/guide/#cli)

```

File: README.md [hunk 2]
```diff
@@ -61,13 +61,28 @@ pnpm add --save-dev prettier @prettier/plugin-pug
...
+
+#### [API](https://prettier.github.io/plugin-pug/guide/#api)
+```js
+await prettier.format("code", {
+  parser: "pug",
+  plugins: ["prettier-plugin-pug"],
+});
+```
+
+#### [Configuration file](https://prettier.github.io/plugin-pug/guide/#configuration-file) (.prettierrc)
```
Comment (Shinigami92) [inline README.md:81]:
```suggestion
#### [Configuration file](https://prettier.github.io/plugin-pug/guide/#configuration-file) (.prettierrc)

```

File: docs/guide/index.md [hunk 3]
```diff
@@ -85,22 +122,4 @@ Pug code with automatic formatting:
 
 ## Type support in configuration file
```
Comment (Shinigami92) [inline docs/guide/index.md:123]: Does this content section do not have content anymore?

**Comment by Shinigami92** [comment]
You can also please run prettier before doing a commit, that does some stuff for you

**Comment by Permanently** [comment]
> You can also please run prettier before doing a commit, that does some stuff for you

 Apologies, didn't cross my mind - my environment's changed so I'm still reinstalling packages 😓


> Does this content section do not have content anymore?

Re: header, amended that on the latest commit, looks like I won't be doing anything that requires thought after work in the future as I'm clearly pooped hahaha

**Review by Shinigami92 — APPROVED** [review]

**Pull request merged**